react-responsive-modal/package.json

```diff
@@ -47,7 +47,7 @@
   "size-limit": [
     {
       "path": "dist/react-responsive-modal.cjs.production.min.js",
-      "limit": "3.8 KB"
+      "limit": "3.9 KB"
```

Comment:
I think you also need to change the size of the esm bundle line 54 :)

Reply:
Yeah, I'll run the `size-limit` locally and update values accordingly.